backend/backoffice/admin.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+
+from backoffice.models import TalkEntry
+
+
+@admin.register(TalkEntry)
+class TalkEntryAdmin(admin.ModelAdmin):
+    search_fields = ("name", "title")
+    ordering = ("name", "date", "title")

```

Comment:
We can have a chronological view by ordering by datetime only.

```suggestion
    ordering = ("-date",)
```

Reply:
That's right, it's changed